app/models/survey.rb

```diff
@@ -0,0 +1,5 @@
+class Survey < ActiveRecord::Base
```

Comment:
And I would end up rewiring evaluations in the next step anyways. Build assessments then allow a survey to be created based off assessments. Maybe easier just to start working in the survey controller?